staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -798,6 +798,12 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		if len(pod.Status.Message) > 0 {
 			w.Write(LEVEL_0, "Message:\t%s\n", pod.Status.Message)
 		}
+		if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.SeccompProfile != nil {
+			w.Write(LEVEL_0, "SeccompProfile:\t%s\n", pod.Spec.SecurityContext.SeccompProfile.Type)
+			if pod.Spec.SecurityContext.SeccompProfile.Type == corev1.SeccompProfileTypeLocalhost {
+				w.Write(LEVEL_0, "LocalhostProfile:\t%s\n", *pod.Spec.SecurityContext.SeccompProfile.LocalhostProfile)
```

Comment:
The container section on L1767 puts its two lines on two different levels, `LEVEL_2` and `LEVEL_3`.
Should L804 be `LEVEL_1`?

https://github.com/kubernetes/kubernetes/blob/a4dac224e7ef5d4c1a50b5c6a5f89c39f3115381/staging/src/k8s.io/kubectl/pkg/describe/describe.go#L1765-L1767